### PR TITLE
add ovo icon according to guidelines

### PR DESCRIPTION
**Pull request by hakiemaul**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by hakiemaul** [comment]
can't find brand icon guidelines for OVO (Indonesian eWallet provider)

**Comment by hakiemaul** [comment]
@NathanPJF please help review. cannot add reviewer in this PR weirdly..

File: app/assets/images/payment_icons/ovo.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...2.879 1.253 1.779 1.253 3.058 0 1.822-.951 3.264-2.668 4.045zm-3.19-1.323c.615.26 1.705.26 2.32 0 1.45-.607 2.158-2.494 1.508-3.969-.255-.542-.963-1.247-1.508-1.474-.58-.25-1.647-.26-2.239-.044-.591.239-1.276.878-1.566 1.464-.301.618-.325 1.92-.058 2.505.267.564.998 1.29 1.543 1.518zm22.075 1.572c-.743.185-1.937.185-2.657 0-1.055-.26-2.157-1.062-2.737-1.984-.743-1.193-.824-3.058-.174-4.294.754-1.42 2.169-2.32 3.828-2.429 2.1-.141 3.805.759 4.698 2.45.348.684.371.781.371 1.963 0 1.193-.023 1.29-.371 1.963a4.6 4.6 0 0 1-2.958 2.331zm-2.471-1.572c.615.26 1.705.271 2.32 0 .545-.228 1.253-.933 1.508-1.475.673-1.518-.081-3.415-1.59-4.012-.59-.228-1.658-.206-2.238.044-1.114.477-1.728 1.463-1.728 2.721s.614 2.245 1.728 2.722zm-9.593.78l-.035-.737-.383-.032c-.313-.033-.371-.076-.313-.217.44-1.139 1.984-4.337 2.042-4.283.058.065 1.415 3.112 1.925 4.337.035.076-.093.141-.336.163l-.383.032-.035.738-.035.726h3.028v-.412c0-.282-.44-1.388-1.462-3.61-.8-1.757-1.589-3.525-1.763-3.926l-.313-.748-.557.054c-.429.032-.65.12-.893.358-.22.206-.847 1.42-1.81 3.513-.812 1.757-1.484 3.21-1.496 3.231-.023.022-.232.044-.476.055l-.452.032-.035.727-.035.726h3.852l-.035-.726z" id="pi-ovo"/></g></g></g></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/ovo.svg]: Hi @hakiemaul , there are some `id` attributes that are unused. We only need to keep `<title id="pi-ovo">OVO</title>` for accessibility purposes. The rest can be removed.

[alt](https://screenshot.click/21-23-w2mff-zj34o.png)

```suggestion
<svg width="38" height="24" viewBox="0 0 38 24" version="1.1" xmlns="http://www.w3.org/2000/svg" aria-labelledby="pi-ovo" role="img"><title id="pi-ovo">OVO</title><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g fill-rule="nonzero"><path d="M35 0H3a3 3 0 0 0-3 3v18a3 3 0 0 0 3 3h32a3 3 0 0 0 3-3V3a3 3 0 0 0-3-3z" fill="#000" opacity=".07"/><path d="M35 1a2 2 0 0 1 2 2v18a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V3a2 2 0 0 1 2-2h32" fill="#FFF"/><g transform="translate(5 8)" fill="#4C3494"><g transform="matrix(1 0 0 -1 0 9)"><path d="M6.728 8.599c-.638.282-.905.336-1.798.368-1.16.033-1.833-.097-2.622-.53C1.59 8.034.742 7.156.371 6.418.093 5.855.058 5.682.058 4.554.07 3.437.104 3.242.371 2.69.951 1.53 2.134.575 3.387.26c1.775-.444 3.538.011 4.756 1.236.882.879 1.253 1.779 1.253 3.058 0 1.822-.951 3.264-2.668 4.045zm-3.19-1.323c.615.26 1.705.26 2.32 0 1.45-.607 2.158-2.494 1.508-3.969-.255-.542-.963-1.247-1.508-1.474-.58-.25-1.647-.26-2.239-.044-.591.239-1.276.878-1.566 1.464-.301.618-.325 1.92-.058 2.505.267.564.998 1.29 1.543 1.518zm22.075 1.572c-.743.185-1.937.185-2.657 0-1.055-.26-2.157-1.062-2.737-1.984-.743-1.193-.824-3.058-.174-4.294.754-1.42 2.169-2.32 3.828-2.429 2.1-.141 3.805.759 4.698 2.45.348.684.371.781.371 1.963 0 1.193-.023 1.29-.371 1.963a4.6 4.6 0 0 1-2.958 2.331zm-2.471-1.572c.615.26 1.705.271 2.32 0 .545-.228 1.253-.933 1.508-1.475.673-1.518-.081-3.415-1.59-4.012-.59-.228-1.658-.206-2.238.044-1.114.477-1.728 1.463-1.728 2.721s.614 2.245 1.728 2.722zm-9.593.78l-.035-.737-.383-.032c-.313-.033-.371-.076-.313-.217.44-1.139 1.984-4.337 2.042-4.283.058.065 1.415 3.112 1.925 4.337.035.076-.093.141-.336.163l-.383.032-.035.738-.035.726h3.028v-.412c0-.282-.44-1.388-1.462-3.61-.8-1.757-1.589-3.525-1.763-3.926l-.313-.748-.557.054c-.429.032-.65.12-.893.358-.22.206-.847 1.42-1.81 3.513-.812 1.757-1.484 3.21-1.496 3.231-.023.022-.232.044-.476.055l-.452.032-.035.727-.035.726h3.852l-.035-.726z" /></g></g></g></g></svg>
```

**Comment by hakiemaul** [comment]
okay done, thanks for the suggestions

**Comment by hakiemaul** [comment]
when will this be released @tauthomas01 ?

**Comment by tauthomas01** [comment]
When #180 will be merged, I will make a release.

**Pull request merged**